cpp/src/sampling/neighbor_sampling_impl.hpp

```diff
@@ -237,7 +266,7 @@ neighbor_sample_impl(raft::handle_t const& handle,
         handle,
         starting_vertices,
         starting_vertex_labels,
-        raft::device_span<vertex_t const>{level_result_dst_vectors.back().data(),
+        raft::device_span<vertex_t const>{level_result_dst_vectors.back().data(), // define a vector . 
```

Comment:
Are we still passing the sampled vertices for the last edge type only?

Reply:
I think you are looking at old code. I no longer have it in my branch

Reply:
You were right. I didn't commit my changes before pushing. All the concerns you had are addressed in my last commit